### ISSUE TITLE
Component level visibility has no effect on child elements

### ISSUE THREAD

**Issue by keresztg** [KlausSchaefers/vue-low-code#6]
Hi!

If I make a component invisible its children are still rendered in Full Mode. They will disappear only if I make them invisible one by one.
This is the base in Figma when everything is visible:
![image](https://user-images.githubusercontent.com/43315663/117133125-1f853f00-ada4-11eb-86f4-0cc94f882403.png)
The corresponding JSON created by figma-low-code's download.js:
[app-visible.json.txt](https://github.com/KlausSchaefers/vue-low-code/files/6426682/app-visible.json.txt)

The corresponding JSON when only the component is invisible:
[app-only_component_invisible.json.txt](https://github.com/KlausSchaefers/vue-low-code/files/6426686/app-only_component_invisible.json.txt)

And only in this case will the "child component" text layer become invisible in the Vue.js app:
![image](https://user-images.githubusercontent.com/43315663/117133258-5196a100-ada4-11eb-9997-a58a8808b37b.png)
The corresponding JSON:
[app-text_invisible_aswell.json.txt](https://github.com/KlausSchaefers/vue-low-code/files/6426691/app-text_invisible_aswell.json.txt)

Thanks,
Gábor

**Comment by KlausSchaefers** [comment]
Thanks! I will take a look what is wrong!

**Comment by keresztg** [comment]
I made some more testes and concluded that the above example is a little bit too synthetic. Usually there's a frame/screen above the component which does not effect the problem but results in a different component tree.
So here is the extended case: 
[letter case test.fig.zip](https://github.com/KlausSchaefers/vue-low-code/files/6433825/letter.case.test.fig.zip)

![image](https://user-images.githubusercontent.com/43315663/117284034-b61a3480-ae66-11eb-8d3d-ec1f8a999da7.png)

I played with the visibility of Comp1Instance and the following JSON files were created:
[0-app-all_visible.json.txt](https://github.com/KlausSchaefers/vue-low-code/files/6433757/0-app-all_visible.json.txt)
[1-app-instance_invisible.json.txt](https://github.com/KlausSchaefers/vue-low-code/files/6433758/1-app-instance_invisible.json.txt)
[2-app-instance_children_invisible.json.txt](https://github.com/KlausSchaefers/vue-low-code/files/6433759/2-app-instance_children_invisible.json.txt)

Also looked around `FigmaService.parseElement` method and it seems to me that parsing the child elements are probably not necessary if the current element is not visible.
So basically moving the "Go down recursive..." block _inside_ the `(!isIgnored&&!isInsisible)` block solved my particular problem but I'm a little bit unsure whether this covers all the possible cases.

Thanks,
Gábor

**Comment by KlausSchaefers** [comment]
Hi,

can you update to 1.0.8. I believe I have fixed the issue.

Thanks for the help.

**Comment by keresztg** [comment]
Hi!

I confirm that version 1.0.8 solves my base problem.

However fiddling with my first synthetic example I have observed that making a frame/screen invisible has no effect on its child components.
Are the top level objects handled elsewhere? Should I create a new issue for this case?

Thanks,
Gábor

**Comment by keresztg** [comment]
Hi!

Yes, version 1.0.9 solved that problem as well.
And I would not call that a bug. Call it a missing feature instead. ;)

Thank,
Gábor
